src/recoils/totalProductsPriceAtoms.ts

```diff
@@ -0,0 +1,19 @@
+import { selector, useRecoilValue } from 'recoil';
+import { checkedCartItemsSelector } from './cartAtoms';
+
+import { RECOIL_KEY } from '@constants/index';
+
+const totalProductsPriceSelector = selector({
+  key: RECOIL_KEY.TOTAL_PRODUCT_PRICE_VALUE,
+  get: ({ get }) => {
+    const checkedCartItems = get(checkedCartItemsSelector);
+
+    const totalProductsPrice = checkedCartItems.reduce((totalPrice, item) => {
+      return totalPrice + item.quantity * item.product.price;
+    }, 0);
+
+    return totalProductsPrice;
```

Comment:
```suggestion
    return checkedCartItems.reduce((totalPrice, { quantity, product }) => totalPrice + quantity * product.price, 0);
```

함수 네이밍이 좋아서 이정도면 충분하지 않을까요?

Reply:
변수에 한번 담아주는 것으로 어떤 것이 반환되는지 쉽게 파악할 수 있도록 만들고 싶었습니다.
포코의 말대로 함수 이름에서 이미 파악이 가능하니, 이 경우에는 포코의 코드처럼 작성해도 전혀 문제가 없어보이네요🤩